src/snowflake/connector/auth_keypair.py

```diff
@@ -45,6 +44,10 @@ def __init__(self, private_key):
         self._private_key = private_key
         self._jwt_token = ""
         self._jwt_token_exp = 0
+        self._lifetime = timedelta(seconds=120)
```

Comment:
I'd rather make this an argument for the `__init__` function. Could we do that instead?
It could be something like:
```
def __init__(self, private_key, lifetime_in_seconds: int = 120):
...
self._lifetime = timedelta(seconds=lifetime_in_seconds)
```

Reply:
Makes sense! Added